CHANGELOG.md

```diff
@@ -10,8 +10,14 @@ IMPORTANT:
 You should stop all containers and plugins **BEFORE** upgrading to Docker CE 17.12.
 See related PR: [moby/moby#35812](https://github.com/moby/moby/pull/35812) 
 
-## 17.12.0-ce (2017-12-DD)
+## 17.12.0-ce (2017-12-27)
 
+## Known Issues
+* AWS logs batch size calculation [moby/moby#35726](https://github.com/moby/moby/pull/35726)
```

Comment:
Nit: newline after headings

Reply:
I can't seem to comment on this because it wasn't changed in this PR, but the admontion in line 8++ should be written as a real admonition:

```
> **Important**: <blah blah>
> More lines
{:.important}
```

Also s/should/must